bin/doctest

```diff
@@ -14,9 +14,15 @@ set -f ; shopt -u nullglob
 prefix="$(get comment-prefix)"
 opening="$(get opening-delimiter)"
 closing="$(get closing-delimiter)"
+module="$(get module-type)"
+
+if [[ "$module" == esm ]] && [[ "$(node_major_version)" -lt 9 ]] ; then
+  echo 'Skipping ESM doctests on Node versions before 9.0.0'
```

Comment:
```suggestion
  echo 'Skipping ESM doctests on Node.js version less than v9.0.0' >&2
```

Rationale:

  - [Node.js][1] seems to be the official project name.
  - The message should refer to a single run rather than to the policy, so _version_ rather than _versions_.
  - It's imprecise to refer to versions released _before_ a particular version, as it implies that all versions released after that version are supported. v8.15.1, for example, was released after v9.0.0 but does not support ESM.
  - The output should go to standard error for compatibility with scripts which process standard output. Although I don't recommend it, `npm run doctest | tail -n +6 | tr -d $'\n' | wc -c` could be used to report the number of doctests run.


[1]: https://en.wikipedia.org/wiki/Node.js

Reply:
Great points all of them! :D